### DISCUSSION TITLE
Local Backup Folder Configuration?

### DISCUSSION THREAD

**Discussion by bigtalljv** [Staubgeborener/klipper-backup#119] [Q&A] [answered]
hi, is it possible to specify a different local backup folder?  rather than putting the backup files in /config I'd like to make a config/Backups folder.  
Thanks

**Comment by Staubgeborener** [comment]
First of all: The default folder is `config_backup` which is located under `~` and **not** `/config`.

Second: This is not possible. Currently, `config_backup` is a [hardcoded value](https://github.com/Staubgeborener/Klipper-Backup/blob/d241976b28e1a327f8d0a0c050044f07bfcf1e02/script.sh#L22) inside the code. You could simply adjust the line manually. But this would lead to a faulty repository in moonraker (which is no big deal tbh) and would be overwritten with every update (which is a big deal).

At the time, we found that debugging any problems was easier if this section was not optional. I think we once had an issue where the topic was clarified, unfortunately I can't find it ad hoc. Most users just want a working printer and have no knowledge of Unix. If you then say “delete your config folder, wherever it is” this immediately leads to confusion for most people. Theoretically this would be solved by the `--fix` parameter, because it works with the `$backup_path` variable to stay flexible.

I mean, I could also leave the value optional, so that it can be chosen with the `.env` file and if no value is specified, then `config_backup` is used as a default value. I just need to check if there are any dependencies that I'm not thinking of right now. But as already mentioned, I think there were other reasons that spoke against it, but I can't find the corresponding issue right now.

Is there a specific reason why you want to do this?

**Comment by Staubgeborener** [comment, marked as answer]
Memo to myself: [this is the reason](https://github.com/Staubgeborener/Klipper-Backup/issues/120), why I will not support this idea.
